backend/src/test/scala/com/softwaremill/bootzooka/test/TestClient.scala

```diff
@@ -0,0 +1,86 @@
+package com.softwaremill.bootzooka.test
+
+import cats.effect.IO
+import com.softwaremill.bootzooka.passwordreset.PasswordResetApi.{ForgotPassword_IN, PasswordReset_IN}
+import com.softwaremill.bootzooka.user.UserApi._
+import io.circe.generic.auto._
+import io.circe.syntax.EncoderOps
+import sttp.client3
+import sttp.client3.{Response, SttpBackend, UriContext, basicRequest}
+
+import scala.util.Random
+
+class TestClient(backend: SttpBackend[IO, Any]) extends TestSupport {
```

Comment:
the only thing I'm wondering about is the naming - I think `Requests` might be better. It expresses the fact that the methods inside send a request with the given name/functionality. What do you think? `TestClient` is quite generic